frontend/src/components/common/checkbox/checkbox.tsx

```diff
@@ -1,42 +1,35 @@
 import * as React from 'react';
 import clsx from 'clsx';
-import { useController, Control } from 'react-hook-form';
-import { FormDefaultValue } from 'common/types';
+import { useForm } from 'react-hook-form';
 
 import styles from './styles.module.scss';
 
 interface Props {
   name: string;
+  value: string;
   label: string;
   isDisabled?: boolean;
-  control: Control;
-  defaultValue?: FormDefaultValue;
+  register: ReturnType<typeof useForm>['register'];
```

Comment:
```suggestion
  register: FormRegisterCb;
```
we'll reuse it so put to frontend-common folder

`type FormRegisterCb = UseFormMethods['register'];`